.github/workflows/canister-tests.yml

```diff
@@ -535,6 +535,14 @@ jobs:
         with:
           name: internet_identity_production.wasm
           path: .
+      - run: |
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+          brew update
+          brew install bash
+          echo "/usr/local/bin" >> $GITHUB_PATH
```

Comment:
is this necessary? how come `bash --version` below works already if bash is not in the PATH?

Reply:
Yes, I think so.
`brew` updates the the current `PATH` (I think), but this would be lost in the next step: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-system-path

Reply:
Gotcha, thought so, just checking!